client-src/elements/chromedash-timeline.ts

```diff
@@ -1,26 +1,27 @@
-import {LitElement, css, html, nothing} from 'lit';
-import {showToastMessage} from './utils.js';
-import {SHARED_STYLES} from '../css/shared-css.js';
+import { LitElement, css, html, nothing } from 'lit';
+import { SHARED_STYLES } from '../css/shared-css.js';
+import { showToastMessage } from './utils.js';
+import { customElement, property, state } from 'lit/decorators.js';
+interface Property {
+    [key: string]: any;
+    0: number;
+    1: string;
+}
 
-class ChromedashTimeline extends LitElement {
-  static get properties() {
-    return {
-      type: {type: String},
-      view: {type: String},
-      props: {attribute: false},
-      selectedBucketId: {attribute: false},
-      showAllHistoricalData: {attribute: false},
-    };
-  }
 
-  constructor() {
-    super();
-    this.selectedBucketId = '1';
-    this.showAllHistoricalData = false;
-    this.type = '';
-    this.view = '';
-    this.props = [];
-  }
+declare global {
+    interface Window {
+        google: any;
+    }
+}
+
+@customElement('chromedash-timeline')
+class ChromedashTimeline extends LitElement {
+    @property({ type: String }) type = '';
```

Comment:
Is prettify putting the decorator and field declaration on the same line.  It doesn't need to change in this PR, but I think I prefer having them on separate lines because I have seen that formatting more often, e.g., in the lit.dev examples.